src/lib/viewers/BaseViewer.js

```diff
@@ -97,6 +107,9 @@ class BaseViewer extends EventEmitter {
     /** @property {boolean} - Whether viewer is being used on a touch device */
     hasTouch;
 
+    /** @property {boolean} - Has the viewer retried downloading the content */
+    haveRetried = false;
```

Comment:
Could you be a little more descriptive with this property name? `haveRetried` is pretty vague

Reply:
`haveRetriedContentDownload`?